libfwupdplugin/meson.build

```diff
@@ -236,6 +236,20 @@ library_deps = [
   platform_deps,
 ]
 
+if have_rust
+  fwupdpluginrs = static_library(
+    'fwupdpluginrs',
+    sources: [
+      'fu-crc.rs',
+    ],
+    rust_crate_type: 'staticlib',
+  )
+  rust = import('unstable-rust')
```

Comment:
any minimum meson version we need to consider?

Reply:
this is the meson rust module, not the rustc compiler version. I think in 1.0.0 the rust module went *stable* and I'll see if the import name needs changing.

Reply:
Yeah, I'd be fine bumping the meson version to the project to 1.0.0 as part of this PR.

Reply:
Done.